setup.cfg

```diff
@@ -35,7 +35,7 @@ classifiers =
     Topic :: Utilities
 
 [options]
-python_requires = >=3.6
+python_requires = >=3.7,<3.10
```

Comment:
I personally _hate_ capping dependencies, especially `python_requires`. Partially because this is not what this option was made for - pip "scrolls back" through versions looking for a match, and you never want an older release with a newer Python. It also does not give a nice error message even if you'd always capped it from day one. And when tensor flow updates to support 3.10, this likely will support it automatically. I would not do this.

Reply:
I agree, this is more an effect that dependencies may don't support it. 

no strong opinion